### ISSUE TITLE
Plugin doesn't specify an api version

### ISSUE THREAD

**Issue by emielderckx** [cnaude/MuteManager#10]
```
[09:51:25 WARN]: Initializing Legacy Material Support. Unless you have legacy plugins and/or data this is a bug!
[09:51:28 WARN]: Legacy plugin MuteManager v0.16-51 does not specify an api-version.
```

**Comment by cnaude** [comment]
Should be fixed in 0.16-61.

**Comment by emielderckx** [comment]
You changed line 9 in pom.xml to 1.16.4 but i think 1.16.5 is the lastest. :)
